src/mono/mono/mini/CMakeLists.txt

```diff
@@ -518,13 +518,11 @@ add_custom_command(
   VERBATIM
 )
 
-if (FALSE)
 add_custom_target(GENERATE_MINTOPS_TYPESCRIPT ALL
-  ${Python3_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/genmintops.py "${CMAKE_CURRENT_SOURCE_DIR}/interp/mintops.def" "${CMAKE_CURRENT_SOURCE_DIR}/../wasm/runtime/mintops.ts"
+  ${Python3_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/genmintops.py "${CMAKE_CURRENT_SOURCE_DIR}/interp/mintops.def" "${CMAKE_CURRENT_SOURCE_DIR}/../../wasm/runtime/mintops.ts"
```

Comment:
@kg I kind of want to experiment with generating this file into `artifacts/` and then using [TypeScript's path mapping](https://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping) or maybe its `rootDirs` config setting (same link, further down) to find the `dotnet-genrated/mintops.ts` file.  (ie add a path mapping for `dotnet-generated` to `$(REPO_TOP)/artifacts/blah/blah/blah`).

I'll try to pay around this weekend.  Not 100% confident it'll work with rollup+TS. but maybe...

Reply:
It's kind of working, in principle.  Except in practice I'm getting some weird unrelated error from rollup.  (it didn't load the typescript plugin and thinks `exports.ts` has some invalid JS syntax)

One observation, @kg that I think it true even for your current version of this stuff:  Running `genmintops.py` from cmake is not really necessary, and is kind of wrong:  the cmake build runs _after_ the rollup TS->JS build.

Instead I think we should just run the python script from MSBuild.

I added this to my wasm.proj instead of the cmake custom command:

```xml
  <Target Name="GenerateMintopsTS"
          Inputs="$(MonoProjectRoot)wasm\runtime\genmintops.py;$(MonoProjectRoot)mono\mini\interp\mintops.def"
          Outputs="$(NativeGeneratedFilesDir)\mintops.ts">
      <Exec Command="$(PythonCmd) $(MonoProjectRoot)wasm/runtime/genmintops.py $(MonoProjectRoot)mono/mini/interp/mintops.def $(NativeGeneratedFilesDir)/mintops.ts" />
  </Target>
```

And then added `GenerateMintopsTS` to the `BuildWithRollup` target:

```xml
  <Target Name="BuildWithRollup"
          Inputs="@(_RollupInputs);$(_RollupEnvIntermediateFile)"
          Outputs="$(NativeBinDir).rollup-stamp"
	  DependsOnTargets="WriteRollupEnvToFile;GenerateMintopsTS"
          >
```

Reply:
Will try migrating to msbuild, thanks.